### PR TITLE
Add informations about devices :iphone:

### PR DESCRIPTION
**Pull request by RxnDbr**

### CODE REVIEW

File: user_research/Trashes/trash_recyclingCenter.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # General information about trashes
 
+The main trashes which are collected in RC are [D's I]:
+1. green trashes
+2. bulkies (chair, toys, ...)
+3. wood
```
Comment (vallettea) [inline user_research/Trashes/trash_recyclingCenter.md]: github's markdown interpretor prefers : 

1) ffsrf
2) sfsf

File: user_research/Trashes/trash_recyclingCenter.md [hunk 2]
```diff
@@ -10,43 +15,91 @@ We can talk about trashes in three different ways:
 There are **83 recycling centers** in Gironde
 There are **2 storage center** , where they bury trashes, in Gironde.
 
-**Machines which are not broken** are not re-used and are thrown out with trashes (in Gironde). Only Ustom keep them (they sort them by themselves) and store them in a **"reclyclerie" ( != recycling center)** (same concept as Emmaüs). It is located in Pessac sur Dordogne. [LV's I]
+**Machines which are not broken** are not re-used and are thrown out (in Gironde). Only Ustom keep them (they sort them by themselves) and store them in a **"recyclerie" ( != recycling center)** (same concept as Emmaüs). It is located in Pessac sur Dordogne. [LV's I]
 
 ![Trashes in Gironde](trashes_Gironde.jpg)
 
 There are **3 different ways to throw out trashes** [LV's I]:
 * Door to door/"Porte à porte" (**PAP**) : wast trucks
```
Comment (vallettea) [inline user_research/Trashes/trash_recyclingCenter.md]: "waste truck"

File: user_research/Trashes/trash_recyclingCenter.md [hunk 3]
```diff
@@ -10,43 +15,91 @@ We can talk about trashes in three different ways:
...
 ![Bulky](bulky.png)
 
 **Compost** is currently not collected separately from the others trashes.  [LV's I]
-*[Personal consideration] : That's too bas,  Idea of a new project about compost collection?*
+* *Personal consideration : That's too bad,  Idea of a new project about compost collection?*
 
 (It is not the same organization/distribution in the entire country [LV's I])
 
+Ustom carries on **recycling centers and trashes' collection** [Delphine's Interview]: 
+* In order to sort their trashes, it has been 2 years that Ustom make **people pay for household waste** depending on their weight. People have thrown out less recycling trashes inside non-recycling bins since that decision.
```
Comment (vallettea) [inline user_research/Trashes/trash_recyclingCenter.md]: makes people pay

File: user_research/Trashes/trash_recyclingCenter.md [hunk 4]
```diff
@@ -10,43 +15,91 @@ We can talk about trashes in three different ways:
...
 **Compost** is currently not collected separately from the others trashes.  [LV's I]
-*[Personal consideration] : That's too bas,  Idea of a new project about compost collection?*
+* *Personal consideration : That's too bad,  Idea of a new project about compost collection?*
 
 (It is not the same organization/distribution in the entire country [LV's I])
 
+Ustom carries on **recycling centers and trashes' collection** [Delphine's Interview]: 
+* In order to sort their trashes, it has been 2 years that Ustom make **people pay for household waste** depending on their weight. People have thrown out less recycling trashes inside non-recycling bins since that decision. 
+* There are **12,000 tones of household** trashes in 2014 against **14,000 tones of recycling** trashes whearas household used to be superior before the motivational taxe
+
```
Comment (vallettea) [inline user_research/Trashes/trash_recyclingCenter.md]: tax

File: user_research/Trashes/trash_recyclingCenter.md [hunk 5]
```diff
@@ -10,43 +15,91 @@ We can talk about trashes in three different ways:
...
+Ustom carries on **recycling centers and trashes' collection** [Delphine's Interview]: 
+* In order to sort their trashes, it has been 2 years that Ustom make **people pay for household waste** depending on their weight. People have thrown out less recycling trashes inside non-recycling bins since that decision. 
+* There are **12,000 tones of household** trashes in 2014 against **14,000 tones of recycling** trashes whearas household used to be superior before the motivational taxe
+    
+The **rush hour** of Ustom's RC are at 10 am and 3 pm [D's I]
+
+**Time spend in RC** [D's I]:
+ * Users spend around **5-6 min inside** the RC
+ * During rush hours, people are **waiting aroud 15 minutes** at the entry 
+  * *Personal consideration : Perfect to place an ad about 6element* :+1:
```
Comment (vallettea) [inline user_research/Trashes/trash_recyclingCenter.md]: :+1:

File: user_research/Trashes/trash_recyclingCenter.md [hunk 6]
```diff
@@ -10,43 +15,91 @@ We can talk about trashes in three different ways:
...
+The **rush hour** of Ustom's RC are at 10 am and 3 pm [D's I]
+
+**Time spend in RC** [D's I]:
+ * Users spend around **5-6 min inside** the RC
+ * During rush hours, people are **waiting aroud 15 minutes** at the entry 
+  * *Personal consideration : Perfect to place an ad about 6element* :+1: 
+  
+In summer, bin are usually full and **people must come back at home with trashes** : *"It would be great to have the information in live"* [d's I]
+
+Bins are under-sized (*sous-dimensionnées?*)
```
Comment (vallettea) [inline user_research/Trashes/trash_recyclingCenter.md]: undersized

File: user_research/Trashes/trash_recyclingCenter.md [hunk 7]
```diff
@@ -10,43 +15,91 @@ We can talk about trashes in three different ways:
...
+* come to the RC at the end of the day : around **5 and 6 pm**
+* pay 120€ per year** in Ustom's RC: they can come how many times they wish but are limited to 3m³ per week [D's I]
+* can be farmers (green trashes), workmen (plaster, ...) , entrepreneurs (cardboard, paper, polyester ...) [D's I]
+* A lot of public recycling centers are not made for professionals who are supposed to use **private professional recycling centers**. There are **5-6** in this kind in Gironde. Some local authority accept them in recycling centers but *try* to make them pay. [LV's I]
 
 ![Professional](professional.png)
-* Punctual users: punctual events such as a move or spring clean. [doc LV]
-* Bigger the place we live in is, more trash we can accumulate before throwing out everything at the same time. People living in tiny places can't keep any trash and have to throw them out immediately. [doc LV]
+
+unctual users: punctual events such as a move or spring clean. [doc LV]
```
Comment (vallettea) [inline user_research/Trashes/trash_recyclingCenter.md]: current ?

File: user_research/Trashes/trash_recyclingCenter.md [hunk 8]
```diff
@@ -10,43 +15,91 @@ We can talk about trashes in three different ways:
...
+
+The user which is the most representative is **family's father coming after spring-clean**
+
+There are a lot of **farmers**, wine-makers, ... in Aquitaine so they represent a big part of users [Delphine's interview]
+
+**Urbans** don't come in recycling center, they throw out their trashes on the street. [V's I]
+
+Recycling centers mastered by Ustom guest particular and professional users [D's I]
+* *Visit* :For 100 visitors, there are around **70 particulars for 30 professionals**
+* *Users* : RC count **22,000 particulars for 1,000 professionals**
```
Comment (iOiurson) [inline user_research/Trashes/trash_recyclingCenter.md]: I'm not sure i get the difference between **visitors** and **users**... in this case, how do you explain the huge difference of proportions: 7/10 compared to 22.000 / 23.000 ?

**Comment by iOiurson** [comment]
@RxnDbr, I have just read it with little care (i will read it more carefully when i get more time), but i have a few generic questions:
- What is the process of defining Personas ? (you don't have to answer now, we can discuss it when we see each other)
- Did you update this with the interviews you did in situ ? (i believe this is the case, but i'm just checking)
- Could you take a moment and look for english mistakes ? From what i have seen, there are quite a lot ^^ no pressure :p

File: user_research/Trashes/trash_recyclingCenter.md (hunk 8 above)
Comment (RxnDbr) [inline user_research/Trashes/trash_recyclingCenter.md]: The difference is about visit frequency : there are 1/22 of RC users who are professionals but for 100 visits (it's a mean) , we can count 30% of professionals. So there are not a lot of professionals registered in comparison with private individuals but they come more often. Am I clear ? How do you think I should rephrase it?

**Comment by RxnDbr** [comment]
@iOiurson 
* Personas : just to be sure : have you read 6element/user_research/Personas/Personas_ScenariosUserTest_Theory.md ? 
 * If you did, what do you think I should add?
* I commited updates on my PC but I did not push it yet : I have probably forgotten some stuff
* I will check my english but I think I will need help because I'm not so good at it ^^"

Thank you for your comments, it's cool to have a feed back =)

File: user_research/Device/digitalData.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
+# 2012 AEC![devices](../Pictures/Devices.png)
```
Comment (iOiurson) [inline user_research/Device/digitalData.md]: What is AEC ?

File: user_research/Device/digitalData.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
...
+![ageInternet](../Pictures/ageAppropriation.png)
+![ageSmartphone](../Pictures/smartphoneAge.png)
+![mobinautes](../Pictures/mobinautes.png)
+# 2011 CREDOC
```
Comment (iOiurson) [inline user_research/Device/digitalData.md]: What is CREDOC ?

File: user_research/Device/digitalData.md [hunk 11]
```diff
@@ -0,0 +1,6 @@
...
+![deviceEvolution](../Pictures/repartitionDeviceFoyerCREDOC.png)
```
Comment (iOiurson) [inline user_research/Device/digitalData.md]: Why this graph contains data up to 2013 when it's supposed to be from 2011 ?

Plus, i understand that according to this 2011 (or 2013 ??) CREDOC poll, smartphone equipment rate is about 39% (for 12+ year olds), and according to AEC 2012 poll, smartphone equipment rate is about 26%. Who should we trust ? Maybe the difference comes from ages considered: AEC could be taking into consideration all population, babies included, which doesn't make sense. If so, CREDOC numbers could be more interesting.

Comment (RxnDbr) [inline user_research/Device/digitalData.md]: Data from CREDOC were in fact published in 2014 (June). I am sorry for the mistake.

Smartphone equipment is about 29% for CREDOC, 39% is percentage in 2013. The gap of 3% can be explained more easily =)
So you want me to send you the entires files on you mailbox or that I check samples' informations?

File: user_research/Device/digitalData.md (hunk 9 above)
Comment (RxnDbr) [inline user_research/Device/digitalData.md]: [Agence Aquinaine du numérique](http://www.aecom.org/Vous-informer/Publications/Diagnostic_AEC/Diagnostic-2012-de-l-Aquitaine-numerique)

File: user_research/Device/digitalData.md (hunk 10 above)
Comment (RxnDbr) [inline user_research/Device/digitalData.md]: [CREDOC](http://www.credoc.fr/)  is the "Centre de Recherche pour l'étude et l'observation des conditions de vie".

I will updates this file with links and a little comment about the data.

CREDOC data come from a study titled "ÉVOLUTIONS DU COMPORTEMENT DES FRANÇAIS FACE AU DEVELOPPEMENT DE L’ECONOMIE CIRCULAIRE".

So the gap between data can be also explained by the difference between population : Credoc observed french people in general whereas AEC observed people living in Aquitaine

**Pull request merged**